src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,212 @@
+import { createActions } from '../';
+import { isFSA } from 'flux-standard-action';
+
+describe('createActions', () => {
+  describe('createActions(actionMap)', () => {
+    it('returns an object with actions for each key', () => {
+      const a = createActions({
```

Comment:
Naming; can we avoid single-letter variables? 

Again, this is an example, but even `actions` would be better and more descriptive than simply `a`.